### ISSUE TITLE
Use go-kit/log for logging 

### ISSUE THREAD

**Issue by metalmatze** [prometheus-community/pro-bing#1]
This would fit the rest of the Prometheus ecosystem better

**Comment by SuperQ** [comment]
As a library, I prefer that we have a logging hook that is compatible with the stdlib logger.

**Comment by metalmatze** [comment]
Makes sense. My only concern with these is that they aren't structured, but don't have that strong of an opinion either.

**Comment by SuperQ** [comment]
There's now https://pkg.go.dev/golang.org/x/exp/slog, which is supposed to be in 1.21 stdlib.
